### PR TITLE
Add Spring Boot to language & frameworks

### PR DESCRIPTION
**Pull request by petretiandrea**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/languages-and-frameworks/spring-boot.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Key Features
+
+- **Great community support**: the large community behind spring boot and Pivotal itself allow for ready-to-use integrations for most commonly used technologies.
+
+- **Spring Cloud**: provides tools for developers to quickly build some of the common patterns in distributed systems. Coordination of distributed systems leads to boiler plate patterns, and using Spring Cloud developers can quickly stand up services and applications that implement those patterns.
+[Spring Cloud](https://spring.io/projects/spring-cloud)
+
+- **Spring Data**: the mission is to provide a familiar and consistent, Spring-based programming model for data access while still retaining the special traits of the underlying data store. It makes it easy to use data access technologies, relational and non-relational databases, map-reduce frameworks, and cloud-based data services. 
+[Spring Data](https://spring.io/projects/spring-data)
```
Comment (gunzip) [inline radar/languages-and-frameworks/spring-boot.md:24]: To decide if we want to adopt these as well, these other technologies should be categorized in separate, distinct entries.

File: radar/languages-and-frameworks/spring-boot.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Our use cases
+- [File Reporter](https://github.com/pagopa/rtd-ms-file-reporter): microservice responsible to provide an aggregated view of files sent and processed by TAE system. Spring Cloud and Spring Data are respectively used to integrate a Kafka consumer and to provide access to a mongo database. By using Spring Framework the business logic which use these two integration are unaware of underlying technology and can be easily changed while the interfaces provided by Spring keep stable.
```
Comment (gunzip) [inline radar/languages-and-frameworks/spring-boot.md:27]: When utilizing acronyms, such as TAE, bear in mind that not all readers may be familiar with these abbreviations.

**Review by gunzip — CHANGES_REQUESTED** [review]

File: radar/languages-and-frameworks/spring-boot.md (hunk 1 above)
Comment (petretiandrea) [reply radar/languages-and-frameworks/spring-boot.md:24]: I'm agree, it's better to move it as distinct entries

File: radar/languages-and-frameworks/spring-boot.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+### Key Features
+
+- **Great community support**: the large community behind spring boot and Pivotal itself allow for ready-to-use integrations for most commonly used technologies.
+
+- **Production ready**: provide production-ready features such as metrics, health checks, and externalized configuration
+
+- **Fast bootstrapping**: a lot of 'starter' dependencies to simplify your build configuration and make service ready to run
+
+### Our use cases
+- [File Reporter](https://github.com/pagopa/rtd-ms-file-reporter): microservice responsible to provide an aggregated view of a file processing system. Thanks to Spring Boot and it's 'starter' dependencies the microservice expose a REST interface and integrate a kafka consumer in a simple and configurable way.
```
Comment (iwoak) [inline radar/languages-and-frameworks/spring-boot.md:25]: @manuraf @GiovanniMancini could you add some other good example of Spring boot usage?

Comment (sebbalex) [reply radar/languages-and-frameworks/spring-boot.md:25]: In PDV we are using Spring boot framework for three different micro-services:
- https://github.com/pagopa/pdv-ms-tokenizer
- https://github.com/pagopa/pdv-ms-person
- https://github.com/pagopa/pdv-ms-user-registry

Comment (manuraf) [reply radar/languages-and-frameworks/spring-boot.md:25]: you could add there repositories, for now :
* https://github.com/pagopa/selfcare-ms-core
* https://github.com/pagopa/selfcare-ms-party-registry-proxy
* https://github.com/pagopa/selfcare-dashboard-backend

Can you edit this branch?

Comment (gunzip) [reply radar/languages-and-frameworks/spring-boot.md:25]: we can add this link that lists all repos: https://github.com/search?q=org%3Apagopa+path%3A**%2Fpom.xml+spring-boot+&type=code

Comment (petretiandrea) [reply radar/languages-and-frameworks/spring-boot.md:25]: Would you prefer to include a list of microservices or a link to a github search page with filters set?

Comment (gunzip) [reply radar/languages-and-frameworks/spring-boot.md]: @iwoak what about this?

Comment (petretiandrea) [reply radar/languages-and-frameworks/spring-boot.md]: I've added a reference to github search page with filters set. If it's ok we can resolve this conversation

**Review by sebbalex — APPROVED** [review]

**Review by gunzip — COMMENTED** [review]
lgtm

**Review by gunzip — APPROVED** [review]

**Review by iwoak — APPROVED** [review]

**Pull request merged**